lesson04/assignment/css/_settings.variables.scss

```diff
@@ -0,0 +1,28 @@
+// **Mixins**
+// Mixins for opacity
+@mixin opaq {
```

Comment:
These will work, but ideally with a mixin you'd have some value to pass in. For example, you could change opaq to take in a value which outputs that value of opacity, so you'd only need 1 mixin that you can call to get the opacity to either 1 or 0 depending on the passed in value. Same with the position ones, could take in a string that returns that as the `position` value.